CMakeLists.txt

```diff
@@ -713,12 +713,22 @@ if(WIN32)
 	endif()
 	if(ENABLE_LAUNCHER)
 		set(CPACK_PACKAGE_EXECUTABLES "VCMI_launcher;VCMI")
-		set(CPACK_NSIS_EXTRA_INSTALL_COMMANDS " CreateShortCut \\\"$DESKTOP\\\\VCMI.lnk\\\" \\\"$INSTDIR\\\\VCMI_launcher.exe\\\"")
+		set(CPACK_NSIS_EXTRA_INSTALL_COMMANDS "
+			CreateShortCut \\\"$DESKTOP\\\\VCMI.lnk\\\" \\\"$INSTDIR\\\\VCMI_launcher.exe\\\"
```

Comment:
make the exe name a param, then you can greatly simplify the if/else

Reply:
How to do this in cmake? simple do do a set before and use $variablename?

Reply:
if you don't want to read cmake docs / some tutorial (although I think it's a good opportunity to read some basics), there's a bunch of such examples in our CML. you declare variables with `set(varname value)` and use it inside a string as `${varname}`

Reply:
You mean like this?

Reply:
yep, looks right. Well done!